### PR TITLE
Fix webui error remove other user new

### PR DESCRIPTION
**Pull request by hascar**
Removed second title and double id in message of the error remove calculation

### CODE REVIEW

File: openquake/server/static/js/engine.js [hunk 1]
```diff
@@ -195,15 +195,15 @@
                                     type: "POST",
                                     error: function (jqXHR, textStatus, errorThrown) {
                                         if (jqXHR.status == 403) {
-                                            diaerror.show(false, "Error", "Calculation:<br><b>(" + calc_id + ") " + calc_desc + "</b> " + JSON.parse(jqXHR.responseText).error);
+                                            diaerror.show(false, "Error", "" + JSON.parse(jqXHR.responseText).error);
```
Comment (micheles) [inline openquake/server/static/js/engine.js]: Why the "" ?

File: openquake/server/static/js/engine.js [hunk 2]
```diff
@@ -195,15 +195,15 @@
...
                                         }
                                     },
                                     success: function(data, textStatus, jqXHR) {
                                         err = data.error;
                                         if(!err) {
                                             err = "removed.";
                                         }
-                                        diaerror.show(false, "Calculation removed", "Calculation:<br><b>(" + calc_id + ") " + calc_desc + "</b> " + err );
+                                        diaerror.show(false, "Calculation removed", "The calculation:<br><b>(" + calc_id + ") " + calc_desc + "</b> " + err );
```
Comment (daniviga) [inline openquake/server/static/js/engine.js]: I would let @micheles comment, but

```
The calculation:
(5) Scenario Calculation with Simple Fault Rupture removed.
```

sounds bad.

I would say
```
The calculation:
(5) Scenario Calculation with Simple Fault Rupture has been removed.
```
or
```
Calculation (5) Scenario Calculation with Simple Fault Rupture removed.
```

File: openquake/server/static/js/engine.js (hunk 1 above)
Comment (daniviga) [inline openquake/server/static/js/engine.js]: `"" +` seems useless here

Comment (hascar) [inline openquake/server/static/js/engine.js]: Fixed here: https://github.com/gem/oq-engine/pull/2918/commits/18f60a121d98c6ce1669365da252aea3cf889264

Comment (hascar) [inline openquake/server/static/js/engine.js]: Fixed here: https://github.com/gem/oq-engine/pull/2918/commits/18f60a121d98c6ce1669365da252aea3cf889264

File: openquake/server/static/js/engine.js [hunk 3]
```diff
@@ -195,15 +195,15 @@
...
                                         }
                                     },
                                     success: function(data, textStatus, jqXHR) {
                                         err = data.error;
                                         if(!err) {
-                                            err = "removed.";
+                                            err = "has been removed.";
                                         }
-                                        diaerror.show(false, "Calculation removed", "Calculation:<br><b>(" + calc_id + ") " + calc_desc + "</b> " + err );
+                                        diaerror.show(false, "Calculation removed", "The calculation:<br><b>(" + calc_id + ") " + calc_desc + "</b> " + err );
```
Comment (micheles) [inline openquake/server/static/js/engine.js]: Please change `The calculation:` -> `The calculation`

**Comment by micheles** [comment]
LGTM

**Pull request merged**